packages/config/src/layer2s/common/stages/getStage.ts

```diff
@@ -123,17 +58,18 @@ export const getStage = createGetStage({
     items: {
       proofSystemOverriddenOnlyInCaseOfABug: {
         positive:
-          'Proof system can be overridden only if a bug is detected by the Ethereum L1 contracts.',
-        negative:
-          'Proof system can be overridden for reasons other than a bug detected by the Ethereum L1 contracts.',
+          'The Security Council is limited to acting solely on adjudicable soundness errors.',
+        negative: `The Security Council's actions are not confined to adjudicable soundness errors.`,
       },
-      fraudProofSystemMustBePermissionless: {
+      fraudProofSystemIsPermissionless: {
         positive: 'The fraud proof system is permissionless.',
-        negative: 'The fraud proof system is permissioned.',
+        negative: 'Fraud proof submission is open to everyone.',
       },
       delayWith30DExitWindow: {
-        positive: 'Users have at least 30 days to exit the system.',
-        negative: 'Users have less than 30 days to exit the system.',
+        positive:
+          'In case of an unwanted upgrade by actors more centralized than a Security Council, users have at least 30d to exit.',
```

Comment:
```suggestion
          'In case of an unwanted upgrade by actors other than a Security Council, users have at least 30d to exit.',
```

Reply:
this part was written by Luca, lets live it for now